android/sdk/src/main/java/com/taobao/weex/ui/component/WXComponent.java

```diff
@@ -175,6 +175,7 @@
   public boolean mIsAddElementToTree = false;
   //for fix element case
   public int interactionAbsoluteX=0,interactionAbsoluteY=0;
+  public boolean mHasAddFocusListener = false;
```

Comment:
Sorry for the delay.

Make `mHasAddFocusListener` to private, thanks.

Reply:
done